src/services/railcontent.js

```diff
@@ -239,6 +239,36 @@ export async function fetchContentPageUserData(contentId) {
     }
 }
 
+/**
+ * Fetches the ID and Type of the piece of content that would be the next one for the user
+ *
+ * @param {int} contentId - The id of the parent (method, level, or course) piece of content.
+ * @returns {Promise<Object|null>} - Returns and Object with the id and type of the next piece of content if found, otherwise null.
+ */
+export async function fetchNextContentDataForParent(contentId) {
+    let url = `/content/${contentId}/next/${globalConfig.railcontentConfig.userId}`;
+    const headers = {
```

Comment:
To avoid code duplication (the other methods should also use the fetch handler)

```suggestion
       return fetchHandler(url); 
```